### ISSUE TITLE
Links disappear on landing grid template

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#269]
Whenever I have a page with the landing grid template enabled (I believe it's landing grid, but it might be content grid) like the [locations page](https://www.jccmi.edu/locations/) and open it up in Wordpress and then publish or save the links are always removed. For instance when I open up the locations page there are links to all of the campuses that are removed when I open the page up in worpdress.

**Comment by d4mation** [comment]
That's because the links are not a part of the Page Content, but actually a part of the Page Template.

![image](https://user-images.githubusercontent.com/7770631/47155559-06364900-d2b3-11e8-9c2e-2237e25b470e.png)

Anything that you place in the Content Editor will show above the Grid of links on that page. Editing the Page and then Saving new Content won't overwrite the Grid as the Grid itself is controlled by these fields further down the Edit Screen: 

![image](https://user-images.githubusercontent.com/7770631/47155711-59a89700-d2b3-11e8-9cb3-31165d6a48cc.png)

**Comment by banksashleym** [comment]
Yes, every time I open up the page to edit in wordpress it resets the "page link" drop down so that that pages I had linked to previously are no longer linked. For instance, on the locations page, each box should have a link to each of the campus location, but since I opened up the page in wordpress the links in the Page Link area now just say "Type to search" and then when I publish there are no longer any links when I hover over the boxes. Anytime I have to change any content on a page that has the landing grid enabled I have to go and relink all of these links in the landing grid area as well. Thanks!
![capture](https://user-images.githubusercontent.com/15264315/47156058-40541a80-d2b4-11e8-9849-3da748b1217b.PNG)

**Comment by d4mation** [comment]
Oh, that's odd. I'll take a closer look at this!

**Comment by banksashleym** [comment]
Thank you!

**Comment by d4mation** [comment]
I just pushed a fix for this to Staging.

https://jacksoncollege.staging.wpengine.com/wp-admin/post.php?post=5511&action=edit

I also added a little "Loading..." text to it while it was still retrieving the options from the database as it was a little slow for me on my local copy of the site, but that doesn't seem to be an issue on Staging so you only see that for a brief second before it finishes.

Let me know if it all looks good!
